### PR TITLE
PDA-142 Apply entando-ui theme

### PR DESCRIPTION
**Pull request by raxkaynan**
* Applies entando-ui theme to PDA
* Updates usage of headings
* Implements PDA-110 and PDA-111
* Updates test snapshots

### CODE REVIEW

**Comment by entando-service** [comment]
All is well

**Comment by entando-service** [comment]
All is well

**Comment by entando-service** [comment]
All is well

**Comment by entando-service** [comment]
All is well

File: widgets/src/components/SummaryCard/SummaryCardContainer.js [hunk 1]
```diff
@@ -223,4 +223,12 @@ SummaryCard.defaultProps = {
   frameId: '',
 };
 
-export default withStyles(styles)(SummaryCard);
+const StyledSummaryCard = withStyles(styles)(SummaryCard);
+
+const SummaryCardContainer = () => (
```
Comment (jeffgo10) [inline widgets/src/components/SummaryCard/SummaryCardContainer.js]: pls forward the props

**Comment by entando-service** [comment]
All is well

**Review by jeffgo10 — APPROVED** [review]
all good

**Pull request merged**